### PR TITLE
Fix: Modal border radius and locked scrollable

### PR DESCRIPTION
**Pull request by ninariccimarie**

### CODE REVIEW

File: src/@next/Modal/ModalStyle.ts [hunk 1]
```diff
@@ -39,7 +40,7 @@ export const StyledModalHeader = styled.div`
   align-items: center;
   padding: ${space16} 20px;
   box-shadow: inset 0px -1px 0px ${Neutral.B95};
-  border-radius: ${borderRadius8} 0;
+  border-radius: ${borderRadius8} ${borderRadius8} 0 0;
```
Comment (ninariccimarie) [inline src/@next/Modal/ModalStyle.ts:43]: fixing these Screenshot 2023-04-03 at 12 16 05 PM

File: src/@next/Modal/ModalStyle.ts [hunk 2]
```diff
@@ -76,7 +77,7 @@ export const StyledModalActions = styled.div`
   padding: ${space16} 20px;
   gap: 10px;
   box-shadow: inset 0px 1px 0px ${Neutral.B95};
-  border-radius: 0 ${borderRadius8};
+  border-radius: 0 0 ${borderRadius8} ${borderRadius8};
```
Comment (ninariccimarie) [inline src/@next/Modal/ModalStyle.ts:80]: fixing these Screenshot 2023-04-03 at 12 16 05 PM

Comment (davidzzz) [reply src/@next/Modal/ModalStyle.ts:80]: what will the design look like if the action section below is not applied?

File: src/@next/Modal/ModalStyle.ts [hunk 3]
```diff
@@ -31,6 +31,7 @@ export const StyledModalContainer = styled.div`
   max-width: calc(100vw - 32px);
   height: fit-content;
   max-height: calc(100vh - 96px);
+  overflow: hidden;
```
Comment (ninariccimarie) [inline src/@next/Modal/ModalStyle.ts:34]: fixes overflow StyledModalContent square edges to always use StyledModalContainer border-radius of 8px

File: src/@next/Modal/ModalStyle.ts (hunk 2 above)
Comment (ninariccimarie) [reply src/@next/Modal/ModalStyle.ts:80]: @davidzzz this is an example https://aries.glints.design/?path=/story/next-modal--with-large-content

**Review by davidzzz — APPROVED** [review]

**Pull request merged**